styles/mixins/_lists.scss

```diff
@@ -3,6 +3,8 @@ Lists mixins - Co-op Front-end Toolkit
 ===============================================*/
 
 @mixin list {
+  @include font-size('body');
+  font-family: $regular;
```

Comment:
Is the `font-family` rule necessary here? seems a bit too opinionated compared to the treatment of other common elements (`p`, etc)

Reply:
This turned into a rabbit hole - but I think I've nailed it.